From 609cc0cc100aa1070d97b405273373682da0e270 Mon Sep 17 00:00:00 2001
From: Jan Friesse <jfriesse@redhat.com>
Date: Fri, 24 Feb 2017 16:23:50 +0100
Subject: [PATCH] Logsys: Change logsys syslog_priority priority

LibQB adds default "*" syslog filter so we have to set syslog_priority
as low as possible so filters applied later in
_logsys_config_apply_per_file takes effect.

Signed-off-by: Jan Friesse <jfriesse@redhat.com>
(cherry picked from commit 8b6bd86a55b8bda9f3a8ff67bdff908263976fa3)
---
 exec/main.c |    7 ++++++-
 1 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/exec/main.c b/exec/main.c
index ca54a47..5126af5 100644
--- a/exec/main.c
+++ b/exec/main.c
@@ -128,10 +128,15 @@
 #define IPC_LOGSYS_SIZE			8192*128
 #endif
 
+/*
+ * LibQB adds default "*" syslog filter so we have to set syslog_priority as low
+ * as possible so filters applied later in _logsys_config_apply_per_file takes
+ * effect.
+ */
 LOGSYS_DECLARE_SYSTEM ("corosync",
 	LOGSYS_MODE_OUTPUT_STDERR | LOGSYS_MODE_OUTPUT_SYSLOG,
 	LOG_DAEMON,
-	LOG_INFO);
+	LOG_EMERG);
 
 LOGSYS_DECLARE_SUBSYS ("MAIN");
 
-- 
1.7.1

